modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeTest.ts

```diff
@@ -69,4 +80,24 @@ describe('browser.tinymce.themes.silver.editor.sizing.ResizeTTest', () => {
     resizeToPos(300, 500, 550, 500);
     assertEditorSize(container, 500, 500);
   });
+
+  it('TINY-4823: can be resized via the keyboard', () => {
+    const editor = hook.editor();
+    const container = TinyDom.container(editor);
+
+    FocusTools.setFocus(SugarBody.body(), '.tox-statusbar__resize-handle');
+
+    // Make it larger
+    for (let i = 0; i < 3; ++i) {
+      TinyUiActions.keystroke(editor, VK.RIGHT);
```

Comment:
Should this be using `Keys` from Agar instead? That's what's normally used here instead of `VK`.